tools/munin/minion

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env perl
```

Comment:
IMHO tools are for developing openQA. How about "contrib/"?

Reply:
I moved the script to `contrib/munin/`